README.md

```diff
@@ -71,17 +71,26 @@ The environment _only will be visible for the lsp server_, not for other extensi
 
 ### Downloaded binaries
 
-This extension will download `haskell-language-server` binaries either via an internal ghcup (it will download it automaticlaly)
-or via a system ghcup (which must be present), unless you set the config option `haskell.manageHLS` to `PATH` (the extension
-will ask you on first start).
+This extension will download `haskell-language-server` binaries and the rest of the toolchain if you selected to use GHCup during
+first start. Check the `haskell.manageHLS` setting.
```

Comment:
I believe it is necessary at the very top of the file to list `GHCup` as a soft-dependency for the automatic management to work.

Reply:
Yes, someone has to clean up the README after this PR.